### PR TITLE
Building and run unikernels with ops

### LINKED ISSUE #4390: Building and Run Unikernels with OPS
### Topic Suggestion
Building and Run Unikernels with OPS 
#### Proposed title of article
Building and Run Unikernels with OPS 

#### Proposed article introduction
Because of their size, security, and performance, unikernels have long been hailed as the next generation of cloud architecture, allowing megatrends like microservices and serverless to thrive while combatting the never-ending barrage of data breaches, cryptojacking, and other issues. However, there's a major issue. Developers have been slow to respond. Some outside the ecosystem have blamed bogus issues such as a lack of debugging, but this isn't the case. For example, gdb is frequently used by everyone I know that uses unikernels.

#### Key takeaways 
1 Introduction to OPS
2 Code, build and run Unkernel

#### References
N/A

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by Gladysoby**
#### Formatting and structure checks
- [X] Have you used the correct folder and file structure? 
- [X] Is your article properly formatted in Markdown?
- [X] Have you used our preferred heading structure? 
- [X] Does your article follow any of the suggested structure formats?
- [X] Is your article over 750 words? 
- [X] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [X] Does your article provide enough value and detail about your topic? 
- [X] Can your article be understood by beginners?
- [X] Have you included a hero image that is 600x400 pixels and have the copyright to use it?
      
#### Grammar & plagiarism checks
- [X] Have you spellchecked and grammar checked your article?
- [X] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit.
- [X] Have you checked your article for readability?
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [X] Are your code snippets properly formatted for syntax highlighting
- [X] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [X] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #4390.

### CODE REVIEW

**Comment by hectorkambow** [comment]
@Gladysoby 
Good afternoon and thank you for submitting your article for publishing.
Your article has been entered into our queue and should be reviewed (and published pending any changes) as soon as a peer reviewer is finished with the ones in the queue before it. 🚀
Thank you.

**Comment by hectorkambow** [comment]
@section-engineering-education/peer-reviewers 

If anyone is available to begin a prelim review on this PR.

**Comment by louisefindlay23** [comment]
Hi, @Gladysoby. Please remove extra spaces and add line breaks (to create paragraphs) where needed to get your article in a ready to review state.

**Comment by Gladysoby** [comment]
@louisefindlay23 I have done the changes you requested

File: content/articles/building and run unikernel with OPS/index.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
+
+### Building and Run Unikernels with OPS
```
Comment (louisefindlay23) [inline content/articles/building and run unikernel with OPS/index.md:2]: Folder names can not have spaces: should be `content/articles/building-and-run-unikernel-with-ops`

File: content/articles/building and run unikernel with OPS/index.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+### key takeaways
+
+- [Introduction to Unikernels](#introduction-to-unikernels)
+- [Code, build and run Unikernels](#code-build-and-run-unikernels)
+
+### Introduction to Unikernels
+
+A unikernel is an executable image that can be executed on a hypervisor without the need for a separate working system. The image consists of software code, as well as all the working machine capabilities required by way of that software.
+Unikernels are constructed with the usage of compilers that leverage library working systems, which might be collections of libraries that constitute a working gadget's core capabilities. This lets a unikernel developer selectively encompass those library additives required to make a software painting, with the unikernel code orchestrating those drivers.<br> Traditional running device functions, along with network or report-machine dealings, are compiled into the very last executable on an as-wanted basis.
```
Comment (louisefindlay23) [inline content/articles/building and run unikernel with OPS/index.md:16]: Remove `<br>` tags.

File: content/articles/building and run unikernel with OPS/index.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+Unikernels hаve lоng has been hаiled аs the next рhаse оf сlоud struсture due tо their length, seсurity, аnd рerfоrmаnсe, аllоwing megаtrends like miсrоserviсes аnd serverless tо рersist desрite the never-ending bаrrаge оf dаtа breасhes, сryрtоjасking, аnd оther рrоblеms. There could be a few issues. The construction industry has been sluggish in responding. Some atmospheres have blаmed fake troubles and a loss of debugging, but this isn't the case. Everyone I know who utilizes unikernels, for example, uses gdb regularly.
+
+### key takeaways
+
+- [Introduction to Unikernels](#introduction-to-unikernels)
+- [Code, build and run Unikernels](#code-build-and-run-unikernels)
+
+### Introduction to Unikernels
+
+A unikernel is an executable image that can be executed on a hypervisor without the need for a separate working system. The image consists of software code, as well as all the working machine capabilities required by way of that software.
```
Comment (louisefindlay23) [inline content/articles/building and run unikernel with OPS/index.md:15]: Lines like this should have an additional line break to separate into paragraphs.

**Review by louisefindlay23 — CHANGES_REQUESTED** [review]

**Comment by hectorkambow** [comment]
@Gladysoby 
Good afternoon and thank you for submitting your topic suggestion. As a rule of thumb - please only submit articles (pull requests) with a complete and fully polished article - ready to be published. 

All revisions and edits should be completed on your own forked repo (as to not take up room in the queue). 
 
If it seems this PR is still quite a bit aways from being completed. We may be closing this PR. 

If so you can go ahead and please re-submit a new PR when its ready to be published (and you can add the duplicate label) and ping the reviewer and myself. 

Closing a PR does NOT mean the article can not be published (or fixed) it simply means that the edits, revisions, and fixes will happen outside of the queue.

**Comment by 07BLAIZER** [comment]
@louisefindlay23 hello
I did the changes you requested but you have closed the PR without giving the specific reason

**Comment by louisefindlay23** [comment]
@Gladysoby, you still need to remove the `content/articles/building and run unikernel with OPS/hero.jpg` file because you haven't deleted the old version once you correctly renamed the folder.

**Pull request closed without merging**